nuitka/build/static_src/CompiledCodeHelpers.c

```diff
@@ -541,6 +541,42 @@ PyObject *BUILTIN_LEN(PyObject *value) {
     return PyInt_FromSsize_t(res);
 }
 
+PyObject *BUILTIN_ANY(PyObject *value) {
+    CHECK_OBJECT(value);
+
+    PyObject *it = PyObject_GetIter(value);
+    PyObject *(*iternextfunc)(PyObject *) = *Py_TYPE(it)->tp_iternext;;
+    PyObject *item;
```

Comment:
Move the declaration of item to where it is used. Unlike CPython, we do C99, so this can be at any point, but esp. inside the loop. Bonus effect, using it outside of the loop, which would be likely a bug symptom will be caught.